attributes/default.rb

```diff
@@ -113,6 +113,7 @@
 # ssl
 default['rabbitmq']['ssl'] = false
 default['rabbitmq']['ssl_port'] = 5671
+default['rabbitmq']['ssl_listen_interface'] = nil  # will not be mentioned in config if left nil
```

Comment:
That's the way it works for many attributes in many cookbooks, please leave this comment out.

Reply:
Or, at least move the comment above the line and make it say that the default will make RabbitMQ listen on all interfaces: that may or may not be obvious.

Reply:
Updated. Waiting for CI to complete.

Reply:
The failure looks not being related to changes:
```
 1) rabbitmq::default ubuntu installs the rabbitmq-server deb_package with the default action
```
Will try to re-trigger test.